recipes/example/meta.yaml

```diff
@@ -47,8 +47,6 @@ about:
   # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
   # See https://opensource.org/licenses/alphabetical
   license: MIT
-  # The license_family, i.e. "BSD" if license is "BSD-3-Clause". (optional)
-  license_family: MIT
```

Comment:
Not sure what happened here,  I keep merge from master before pushing.  I put the `master` `recipe/example` back.